debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -234,7 +234,7 @@ public static void j9bcutil_dumpRomMethod(PrintStream out, J9ROMMethodPointer ro
 	 *			-ACC_TRANSIENT
 	 *			-ACC_SYNTHETIC
 	 *			-ACC_ENUM
-	 *			-ACC_STRICT
+	 *			-ACC_STRICT_INIT
```

Comment:
Better write this comment as `(ACC_STRICT | ACC_STRICT_INIT)`. `ACC_STRICT` is still there in old class files.

Reply:
This list is for field flags only. ACC_STRICT was only applicable for methods.